drivers/scheduler/scheduler.go

```diff
@@ -323,6 +329,27 @@ type Driver interface {
 
 	//RecyleNode deletes nodes with given node
 	RecycleNode(n node.Node) error
+
+	// DeletePXPods delete PX pods
+	DeletePXPods(nameSpace string) error
```

Comment:
we already have DeleteTask for that, please use it, no need Portworx specific method

Reply:
@mborodin-px I tried using DeleteTask but how to use it for portworx pods controller by StorageCluster..